### PR TITLE
Bump sane-pdf-report docker image

### PR DESCRIPTION
**Pull request by lmichalevich**
## Related PRs
https://github.com/demisto/dockerfiles/pull/17231

## Related Issues
fixes: [[CRTX-67937](https://jira-hq.paloaltonetworks.local/browse/CRTX-81731)](https://jira-hq.paloaltonetworks.local/browse/CRTX-81731)

### CODE REVIEW

File: Packs/Base/ReleaseNotes/1_32_10.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Scripts
+##### SanePdfReports
+- Updated the Docker image to: *demisto/sane-pdf-reports:1.0.0.62999*.
```
Comment (yanyo87) [inline Packs/Base/ReleaseNotes/1_32_10.md:3]: Please add a short explanation

Comment (yanyo87) [reply Packs/Base/ReleaseNotes/1_32_10.md:3]: add support for sla field ....

Comment (lmichalevich) [reply Packs/Base/ReleaseNotes/1_32_10.md:3]: @amshamah419

Comment (lmichalevich) [reply Packs/Base/ReleaseNotes/1_32_10.md:3]: Updated @amshamah419

**Review by yanyo87 — APPROVED** [review]

**Pull request merged**